parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -360,11 +360,22 @@ impl cumulus_pallet_xcm::Config for Runtime {
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 }
 
+pub type AgentIdOf = HashedDescription<H256, DescribeFamily<DescribeAllTerminal>>;
+
+/// Ethereum control pallet: Type for specifying how a `MultiLocation` can be converted into an `AccountId`. This is used
+/// when determining sovereign accounts for asset transacting
+pub type SovereignAccountOf = (
+	// Relay chain, sibling parachains, and local children
+	LocationToAccountId,
+	// Other nested consensus systems on sibling parachains
+	HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>,
+);
+
```

Comment:
A bit weird we mix these two different ways to derive the sovereign account, I would prefer to remove `LocationToAccountId` and stick only to `HashedDescription` way.

Reply:
I think we need to go this route to remain backwards compatible with existing sovereign accounts on BridgeHub, which use the older `LocationToAccountId` method. These older sovereign are used by other pallets, and so we can't really ask other parachains to maintain two sovereign accounts on BridgeHub.